p2p/proto/event.proto

```diff
@@ -0,0 +1,26 @@
+syntax = "proto3";
+import "p2p/proto/common.proto";
+
+message Event {
+    Felt252  from_address = 1;
+    repeated Felt252 keys = 2;
+    repeated Felt252 data = 3;
+}
+
+message EventsRequest {
+    Iteration iteration = 1;
+}
+
+message Events {
+    repeated Event items = 1;
+}
```

Comment:
We should be able to associate a sequence of events with a transaction, shouldn't it be more like:

```suggestion
message Events {
    repeated TxnEvents items = 1;
}

message TxnEvents {
    Hash txn_hash = 1;
    repeated Event events = 2;
}
```